docs/conceptual/tuningdb.rst

```diff
@@ -29,7 +29,6 @@ Enable this feature using these commands:
 
 .. code:: bash
 
-  export MIOPEN_FIND_MODE=3
```

Comment:
Found that setting find enforce disables find mode inputs. So removing these unnecessary lines.